docs/index.rst

```diff
@@ -37,7 +37,7 @@ Welcome! DCC++ EX is an easy to use, do-it yourself and affordable, open source
 Mission
 #######
 
-Our mission is to make the world of model railroading universally accessible.
+Our mission is to make the world of digital model railroading universally accessible.
```

Comment:
I didn't want to just change your Mission Statement!
This is just a suggestion, that what you're working on is for digital model railroading specifically.
What do you think?